src/Microsoft.AspNet.Mvc.Core/Infrastructure/MvcRouteHandler.cs

```diff
@@ -66,6 +67,7 @@ public VirtualPathData GetVirtualPath(VirtualPathContext context)
             // without affecting something upstream.
             var oldRouteData = context.RouteData;
             var newRouteData = new RouteData(oldRouteData);
+            newRouteData.Values.Remove(AttributeRouting.RouteGroupKey);
```

Comment:
Might want to group this with the next blok and make a omment that this is done to 'simulate' the output of onventional routing.